tool/teleport/common/teleport.go

```diff
@@ -593,6 +641,252 @@ func onCheckHome() error {
 	return nil
 }
 
+// configureDatabaseBootstrapFlags database configure bootstrap flags.
+type configureDatabaseBootstrapFlags struct {
+	config  dbconfigurators.BootstrapFlags
+	confirm bool
+}
+
+// onConfigureDatabaseBootstrap subcommand that bootstraps configuration for
+// database agents.
+func onConfigureDatabaseBootstrap(flags configureDatabaseBootstrapFlags) error {
```

Comment:
Can we please move all these new methods out of `teleport.go` and into its own source file e.g. `configurator.go`?